support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierFirstStepLanding.tsx

```diff
@@ -0,0 +1,163 @@
+} from 'helpers/redux/storeHooks';
+import { navigateWithPageView } from 'helpers/tracking/ophan';
+
+const checkoutBtnStyleOverrides = css`
+	width: 100%;
+	justify-content: center;
+`;
+
+const darkBackgroundContainerMobile = css`
+	display: flex;
+	background: linear-gradient(
+		to bottom,
+		${palette.brand[400]} 441px,
+		${palette.neutral[97]} 441px,
+		${palette.neutral[97]} 100%
+	);
+	${until.tablet} {
+		background-color: ${palette.brand[400]};
+		border-bottom: 1px solid ${palette.brand[600]};
+	}
+	min-height: 480px;
+	${from.desktop} {
+		min-height: 440px;
+	}
+	padding-top: ${space[6]}px;
+`;
+
+const links = [
+	{
+		href: 'https://www.theguardian.com/info/privacy',
+		text: 'Privacy policy',
+		isExternal: true,
+	},
+	{
+		text: 'Privacy settings',
+		onClick: () => {
+			cmp.showPrivacyManager();
+		},
+	},
+	{
+		href: 'https://www.theguardian.com/help/contact-us',
+		text: 'Contact us',
+		isExternal: true,
+	},
+	{
+		href: 'https://www.theguardian.com/help',
+		text: 'Help centre',
+		isExternal: true,
+	},
+];
+
+export function ThreeTierLandingPage(): JSX.Element {
+	const dispatch = useContributionsDispatch();
+	const navigate = useNavigate();
+	const contributionType = useContributionsSelector(getContributionType);
+	const amount = useContributionsSelector(getUserSelectedAmount);
+
+	const { abParticipations } = useContributionsSelector(
+		(state) => state.common,
+	);
+
+	const { countryGroupId } = useContributionsSelector(
+		(state) => state.common.internationalisation,
+	);
+
+	const countrySwitcherProps: CountryGroupSwitcherProps = {
+		countryGroupIds: [
+			GBPCountries,
+			UnitedStates,
+			AUDCountries,
+			EURCountries,
+			NZDCountries,
+			Canada,
+			International,
+		],
+		selectedCountryGroup: countryGroupId,
+		subPath: '/contribute',
+	};
+
+	const proceedToNextStep = useOtherAmountValidation(() => {
+		const destination = `checkout?selected-amount=${amount}&selected-contribution-type=${contributionType.toLowerCase()}`;
+		navigateWithPageView(navigate, destination, abParticipations);
+	}, false);
+
+	useEffect(() => {
+		dispatch(resetValidation());
+	}, []);
+
+	return (
+		<PageScaffold
+			id="supporter-plus-landing"
+			header={
+				<>
+					<Header>
+						<CountrySwitcherContainer>
+							<CountryGroupSwitcher {...countrySwitcherProps} />
+						</CountrySwitcherContainer>
+					</Header>
+				</>
+			}
+			footer={
+				<FooterWithContents>
+					<FooterLinks links={links}></FooterLinks>
+				</FooterWithContents>
+			}
+		>
+			<Container
+				sideBorders
+				topBorder
+				borderColor="rgba(170, 170, 180, 0.5)"
+				cssOverrides={darkBackgroundContainerMobile}
+			>
+				<h1>Support fearless, independent journalism</h1>
+				<ThemeProvider theme={buttonThemeReaderRevenueBrand}>
+					<Button
+						iconSide="left"
+						priority="primary"
+						size="default"
+						cssOverrides={checkoutBtnStyleOverrides}
+						onClick={proceedToNextStep}
+					>
+						Support with XX
```

Comment:
```suggestion
						Support Us
```

Reply:
This button is there at the moment just as a placeholder, customers will not be able to see this as the ab test is turned off. It won't be there when it is as the content will be replaced in a subsequent pr